src/Views/migration.blade.php

```diff
@@ -0,0 +1,40 @@
+{!! $phpTag !!}
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use BuildEmpire\Apartment\ApartmentMigration;
+
+/**
+ * Apartment migration file.
+ *
+ * You must set the table name with the table property, and place it anywhere you reference the table in your schemas.
+ */
+class {{ $migrationName }} extends ApartmentMigration
+{
+    /**
+    * You must use this table variable in your schema migrations as the table name.
+    */
+    protected $table = '';
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function apartmentUp()
+    {
+        Schema::create($this->table, function (Blueprint $table) {
```

Comment:
yes, it's up to the user to enter the $table they want to create, update or set etc. We are checking if it is empty on code run. It's like a basic template for them.